static/styles/portico-signin.css

```diff
@@ -626,7 +626,7 @@ button.login-google-button {
     text-align: left;
 }
 
-.split-view .actions a {
+.split-view .actions a , .back-to-login {
```

Comment:
@amanharitsh123 Have you checked your failing build logs at https://travis-ci.org/zulip/zulip/jobs/283147916? They point out a linter error in your CSS selectors (need to be on spserate lines).